sdk/python/kfp_tekton/compiler/pipeline_utils.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 kubeflow.org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from kfp import dsl
+
+TEKTON_PIPELINE_ANNOTATIONS = ['tekton.dev/output_artifacts', 'tekton.dev/input_artifacts', 'tekton.dev/artifact_bucket',
```

Comment:
nit, can this list be in alphabetical order?

Reply:
done

Reply:
thanks Tommy

/lgtm